AtomicLib/FieldContents.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System.Reflection;
+using System.Xml.Serialization;
+using AtomicLib.XML;
+
+namespace AtomicLib;
+
+[Serializable]
+[XmlRoot("message")]
```

Comment:
The root should be something that makes sense here, so something akin to `FieldContents`